# ImageProcessing.srv

int16 a
---
int16 x_center_offset
int16 y_center_offset
float32 angle_offset
int8 color
